examples/btc-to-evm-fungible-transfer/.env.sample

```diff
@@ -0,0 +1,13 @@
+SYGMA_ENV=testnet
+BLOCKSTREAM_URL="your blockstream url"
+DESTINATION_ADDRESS="your evm destination address"
+DESTINATION_DOMAIN_CHAIN_ID="destination domain chain id"
+RESOURCE_ID="resource id"
+SOURCE_DOMAIN_CAIPID="source domain caip id"
```

Comment:
same for this one `SOURCE_CAIPID`

Reply:
but if anyone is trying this example and going to the shared-config repo to get the data from there, this person will need to be aware that he needs to put here `caipId` and not `chainId` or `domainId`. Being explicit here could provide guidance afaik.

Reply:
wouldn't `SOURCE_CAIPID` be enough? I think of `DOMAIN` as more of an internal sygma keyword

Reply:
this is the issue of reading PR comments first thing in the morning 😄